datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/JSR310Module.java

```diff
@@ -18,6 +18,7 @@
 
 import java.time.*;
 
+import com.fasterxml.jackson.core.json.PackageVersion;
```

Comment:
Please remove this, that is the wrong `PackageVersion`: the real one is generated on this package.
(generated with `./mvnw clean compile`)

Reply:
Actually, I'll go ahead and remove it from pr.